mysql-test/suite/sys_vars/t/innodb_buffer_pool_load_now_basic.test

```diff
@@ -5,6 +5,19 @@
 -- source include/have_innodb.inc
 --source include/not_embedded.inc
 
+# Restart mysqld to ensure that the status variables below do not contain
+# stale value from previous executions of this or other tests. The following
+# race could happen without this restart:
+# 0. innodb_buffer_pool_load_status is 'Buffer pool(s) load completed ...'
+# 1. SET innodb_buffer_pool_load_now = ON
+# (1. does not start executing yet)
+# 2. Wait for innodb_buffer_pool_load_status to become '... completed ...',
+#    the condition is satisfied immediately
```

Comment:
Right, we have to use "-x" option for git cherry-pick going forward:

"       -x
           When recording the commit, append a line that says "(cherry picked from commit ...)" to the original commit message in order to indicate which commit
           this change was cherry-picked from. This is done only for cherry picks without conflicts. Do not use this option if you are cherry-picking from your
           private branch because the information is useless to the recipient. If on the other hand you are cherry-picking between two publicly visible branches
           (e.g. backporting a fix to a maintenance branch for an older release from a development branch), adding this information can be useful."